networklibrary/src/main/java/com/eis/communication/network/Commands/AddResource.java

```diff
@@ -0,0 +1,27 @@
+package com.eis.communication.network.Commands;
+
+import com.eis.communication.network.SMSNetDictionary;
+
+/**
+ * Command to add a resource
+ *
+ * @author Edoardo Raimondi
+ */
+public class AddResource implements Command {
+
+    /*Using strings to be coherent to SMSNetDictionary class*/
+    private String key;
+    private String value;
+    private SMSNetDictionary netDictionary;
+
+    public AddResource(String key, String value, SMSNetDictionary netDictionary){
```

Comment:
Why `SMS`NetDictionary and not a generic `NetworkDictionary`? Why is this restricted to SMS and if this is wanted, why is it in the `network` package (where there should be generic things)?

Reply:
Also missing specifications lol